src/DownloadFormat/DownloadFormat.styles.ts

```diff
@@ -0,0 +1,74 @@
+import styled, { css } from 'styled-components';
+
+interface OptionProps {
+  variant?: string;
+}
+
+export const FormatOption = styled.div<OptionProps>`
+  text-align: center;
+  font-weight: bold;
+  height: 36px;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:first-child {
+    border-radius: 5px 5px 0 0;
+  }
+
+  &:last-child {
+    border-radius: 0 0 5px 5px;
+  }
+
+  &:hover {
+    background-color: ${p => p.theme.colors.lightSkyBlue};
+    color: ${p => p.theme.colors.white};
+  }
+
+  &:active {
+    background-color: ${p => p.theme.colors.darkSkyBlue};
+  }
+
+  &:not(:last-child) {
+    border-bottom: 1px solid ${p => p.theme.colors.gray3};
+  }
+
+  ${p =>
+    p.variant === 'disabled' &&
+    css`
+      &:hover {
+        background-color: ${p.theme.colors.gray3};
+      }
+    `};
+`;
+
+export const FormatList = styled.div`
+  background-color: ${p => p.theme.colors.white};
+  width: 80px;
```

Comment:
I wouldn't take these values from Figma, it doesn't really know what we want. We should try to make things a bit more fluid, like having it fit the width with some padding on either side - then we can rue is all over the place and it will just work.